### ISSUE TITLE
Only after joining the party do you start to synchronize data.

### ISSUE THREAD

**Issue by zxyxz** [tiltedphoques/TiltedEvolution#398]
## Description

The connection to the server does not start syncing and only supports chat and party invitations. You only start syncing data after you join the party.

## Additional context

Each server supports multiple parties, and the public server can operate normally to reduce the occurrence of crash events.

**Comment by RobbeBryssinck** [comment]
This is very vague. Please make a new issue and elaborate on what "syncs" and what doesn't, and what specific issues you're currently having.

**Comment by zxyxz** [comment]
This is a feature  request,not bug .

**Comment by zxyxz** [comment]
> This is very vague. Please make a new issue and elaborate on what "syncs" and what doesn't, and what specific issues you're currently having.

This is a feature request,not bug .

**Comment by Dragonisser** [comment]
> This is a feature request,not bug .

We have no idea what you mean.

**Comment by zxyxz** [comment]
sorry
